manim/mobject/geometry.py

```diff
@@ -406,6 +428,41 @@ def __init__(
         )
 
     def surround(self, mobject, dim_to_match=0, stretch=False, buffer_factor=1.2):
+        """Modifies a circle so that it surrounds a given mobject.
+
+        Parameters
+        ----------
+        mobject : :class:`~.Mobject`
+            The mobject that the circle will be surrounding
```

Comment:
`dim_to_match` and `stretch` are missing in the parameter list

Reply:
As far as im aware, `dim_to_match` doesn't affect the end result. I also assumed that `stretch` does nothing, but it does impact the look of the circle, so I'll add that. Note: `stretch` does not work for lines, no idea why.
![image](https://user-images.githubusercontent.com/70682032/111077908-538d6380-84c9-11eb-8f71-fba118b46d09.png)
just kind of eats the line